include/brod_int.hrl

```diff
@@ -43,7 +43,7 @@
                   | ?undef.
 
 %% Is kafka error code
--define(IS_ERROR(EC), kpro_error_code:is_error(EC)).
+-define(IS_ERROR(EC), EC =/= ?no_error).
```

Comment:
This macro definition is unsafe. Fix:

      -define(IS_ERROR(EC), ((EC) =/= ?no_error)).